### PR TITLE
Adds coding conventions to the bigip_facts module

### PR DESCRIPTION
**Pull request by caphrim007**
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

network/f5/bigip_facts.py
##### ANSIBLE VERSION

```
ansible 2.1.0.0
```
##### SUMMARY

A number of coding conventions have been adopted for new F5 modules
that are in development. To ensure common usage across the modules,
this module needed to be updated to reflect those conventions.

No functional code changes were made.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @caphrim007. This module is going into community review.

We notice that you are one of the maintainers (@mhite, @caphrim007) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by caphrim007** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @caphrim007. Since you are one of the maintainers (@mhite, @caphrim007) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: network/f5/bigip_facts.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
-
-# (c) 2013, Matt Hite <mhite@hotmail.com>
```
Comment (resmo) [inline network/f5/bigip_facts.py:4]: Could you explain why do you remove the copyright?

**Comment by caphrim007** [comment]
@resmo whoops, oversight on my part. Thanks for catching that. updated PR

**Comment by resmo** [comment]
@caphrim007 no problem, thanks!

**Pull request merged**